### DISCUSSION TITLE
How do i add more formats or validations AJV

### DISCUSSION THREAD

**Discussion by JorgeSilva7** [nanoexpress/nanoexpress#175] [General] [answered]
Hi,
i really want to work with nanoexpress, i want to use all the benefits it offers. This is why I want to use the 'email' validation that ajv offers. But i can't use it :( 

On the other hand, I can't get to the swagger url :( 

thanks

**Comment by dalisoft** [comment, marked as answer]
Please use schemator middleware

**Reply by dalisoft** [reply]
I recommend using schemator because of upcoming releases removes swagger and ajv validation from core for making nanoexpress modular

**Reply by JorgeSilva7** [reply]
oooh okey! thanks!
